ft_regressconfound.m

```diff
@@ -94,6 +96,7 @@
 cfg.normalize  = ft_getopt(cfg, 'normalize', 'yes');
 cfg.model      = ft_getopt(cfg, 'model', 'no');
 cfg.statistics = ft_getopt(cfg, 'statistics', 'no');
+cfg.beta       = ft_getopt(cfg, 'beta', 'no');
```

Comment:
The default used to be that beta was kept. Why change that default?

I propose to call the option cfg.keepbeta, in line with other cfg.keepxxx options elsewhere in FT.